test/softdevice_api/testcase_phy_update.cpp

```diff
@@ -102,8 +102,7 @@ TEST_CASE(CREATE_TEST_NAME_AND_TAGS(phy_update, [known_issue][PCA10056][PCA10059
         c->setGapEventCallback([&](const uint16_t eventId, const ble_gap_evt_t *gapEvent) -> bool {
             switch (eventId)
             {
-                case BLE_GAP_EVT_CONNECTED:
-                {
+                case BLE_GAP_EVT_CONNECTED: {
```

Comment:
I see, at some places the curly bracket is at the end of line while at some places it is at the beginning of line. Is it following some standards?

Reply:
The configuration for braces is found here:
https://github.com/NordicSemiconductor/pc-ble-driver/blob/f357dba5adb00ace25c83cede4f4a5f4fcc76538/.clang-format#L22

The AfterCaseLabel is false by default so this is correct. Here are the default values (clang-format --dump-config):

```
BraceWrapping:
  AfterClass:      true
  AfterControlStatement: true
  AfterEnum:       false
  AfterFunction:   true
  AfterNamespace:  false
  AfterObjCDeclaration: false
  AfterStruct:     true
  AfterUnion:      false
  AfterExternBlock: false
  BeforeCatch:     true
  BeforeElse:      true
  IndentBraces:    false
  SplitEmptyFunction: false
  SplitEmptyRecord: false
  SplitEmptyNamespace: false
````